### ISSUE TITLE
Classification unit size

### ISSUE THREAD

**Issue by mrholiday** [mrholiday/LORELEI-Situation-Frame-Pipeline-Year-1#1]
We call the unit for classification "**snippet**".
For training set, for now, each snippet is a document.
For test set, we have two choices: segment-wise or document-wise. For former, each segment (a sentence from XML file) is a test example; for the latter, each whole document is a test example.

- [ ] We want to change the size of snippet for test set first.
- [ ] We want to change the size of snippet for training set, but that has much lower priority.
